semantic/src/themes/default/globals/site.variables

```diff
@@ -325,53 +329,58 @@
 @subtleGradient: linear-gradient(transparent, @transparentBlack);
 
 /* Differentiating Layers */
-@subtleShadow: 0px 1px 2px 0 @borderColor;
-@floatingShadow: 0px 2px 4px 0px rgba(34, 36, 38, 0.12), 0px 2px 10px 0px rgba(34, 36, 38, 0.15);
+@subtleShadow:
+  0px 1px 2px 0 @borderColor
+;
+@floatingShadow:
+  0px 2px 4px 0px rgba(34, 36, 38, 0.12),
+  0px 2px 10px 0px rgba(34, 36, 38, 0.15)
+;
 
 /*******************************
            Power-User
 *******************************/
 
+
 /*-------------------
     Emotive Colors
 --------------------*/
 
 /* Positive */
-@positiveColor: @green;
-@positiveBackgroundColor: #fcfff5;
-@positiveBorderColor: #a3c293;
-@positiveHeaderColor: #1a531b;
-@positiveTextColor: #2c662d;
+@positiveColor           : @green;
+@positiveBackgroundColor : #FCFFF5;
+@positiveBorderColor     : #A3C293;
+@positiveHeaderColor     : #1A531B;
+@positiveTextColor       : #2C662D;
 
 /* Negative */
-@negativeColor: @red;
-@negativeBackgroundColor: #fff6f6;
-@negativeBorderColor: #e0b4b4;
-@negativeHeaderColor: #912d2b;
-@negativeTextColor: #9f3a38;
+@negativeColor           : @red;
+@negativeBackgroundColor : #FFF6F6;
+@negativeBorderColor     : #E0B4B4;
+@negativeHeaderColor     : #912D2B;
+@negativeTextColor       : #9F3A38;
 
 /* Info */
-@infoTextColor: @headersInteractiveLow;
-@infoHeaderColor: @headersInteractiveLow;
-@infoColor: @surfacesLight;
-@infoBoxShadow: @surfacesLight;
-@infoBorderColor: @surfacesLight;
-@infoBackgroundColor: @surfacesLight;
```

Comment:
Yeah, here they are! So you should be able to just copy these over to `messages.variables` now to fix.

```
@infoTextColor: @headersInteractiveLow;
@infoHeaderColor: @headersInteractiveLow;
@infoColor: @surfacesLight;
@infoBoxShadow: @surfacesLight;
@infoBorderColor: @surfacesLight;
@infoBackgroundColor: @surfacesLight;
```

Reply:
Thank you so much for doing this : )